src/Repository/Config/SVNRepositoryConfig.php

```diff
@@ -46,6 +46,8 @@ class SVNRepositoryConfig extends RepositoryConfig {
 		'cache-ttl' => 0,
 		// provider cache file
 		'cache-file' => 'providers.json',
+		// should the server cert be automatically trusted?
+		'trust-cert' => false,
```

Comment:
Seems like you want this on by default rather than having to switch it on for each type of repository – at least since it would create fewer overrides. But I doubt that it would be "proper" to default to a "less secure" mode.